app/uk/gov/hmrc/fileupload/read/envelope/ValidFileTypes.scala

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
This does not have any use anymore if the configuration is done in the conf file. This can be removed?